src/tests/badge.rs

```diff
@@ -350,48 +365,48 @@ fn appveyor_extra_keys() {
     appveyor_attributes.insert(String::from("extra"), String::from("info"));
     badges.insert(String::from("appveyor"), test_badges.appveyor_attributes);
 
-    Badge::update_crate(&conn, &krate, Some(&badges)).unwrap();
-    assert_eq!(krate.badges(&conn).unwrap(), vec![test_badges.appveyor]);
+    krate.update(&badges);
+    assert_eq!(krate.badges(), vec![test_badges.appveyor]);
 }
 
 #[test]
 fn travis_ci_required_keys() {
     // Add a travis ci badge missing a required field
-    let (conn, krate, mut test_badges) = set_up();
+    let (krate, mut test_badges) = set_up();
 
     let mut badges = HashMap::new();
 
     // Repository is a required key
     test_badges.travis_ci_attributes.remove("repository");
     badges.insert(String::from("travis-ci"), test_badges.travis_ci_attributes);
 
-    let invalid_badges = Badge::update_crate(&conn, &krate, Some(&badges)).unwrap();
+    let invalid_badges = krate.update(&badges);
     assert_eq!(invalid_badges.len(), 1);
-    assert_eq!(invalid_badges.first().unwrap(), "travis-ci");
-    assert_eq!(krate.badges(&conn).unwrap(), vec![]);
+    assert_eq!(invalid_badges.first().unwrap(), &"travis-ci");
```

Comment:
Hm, I tried removing these added `&`s in front of the string literals and the tests still compiled and passed? Was there an intermediate state of `BadgeTestCrate.update` where this was necessary maybe? Mind taking them back out again if they're no longer needed?